ios/brave-ios/Sources/Brave/Assets/Interstitial Pages/Pages/NetworkError.html

```diff
@@ -22,7 +22,7 @@
             %error_title%
         </p>
 
-        <p class="description">
+        <div class="description">
```

Comment:
https://html.spec.whatwg.org/dev/grouping-content.html#the-p-element

> List elements (in particular, ol and ul elements) cannot be children of p elements. 

Could someone from ios team please confirm this page is still rendered correctly?

Reply:
tested. renders correctly for me